### PR TITLE
add participants.json with Levels as objects

### LINKED ISSUE #412: Update examples to demonstrate using object for levels in sidecar for TSV
Should implement: https://github.com/bids-standard/bids-specification/pull/1603
Could then be used to test the validator.

I would recommend updating an example that has no `participants.json`

### PR DESCRIPTION
**Pull request by Remi-Gau**
closes #412

### CODE REVIEW

**Comment by Remi-Gau** [comment]
@surchs I may need your help on this

**Review by sappelhoff — COMMENTED** [review]
We are getting this issue:

```
  	1: [ERR] Invalid JSON file. The file is not formatted according the schema. (code: 55 - JSON_SCHEMA_VALIDATION_ERROR)
		./participants.json
			Evidence: ['sex'].Levels['F'] should be string
		./participants.json
			Evidence: ['sex'].Levels['M'] should be string
		./participants.json
			Evidence: ['sex'].TermURL should match format "uri"
```

So we probably need to tell the (legacy JS) validator / the schema that levels as objects are fine (not only string)

And we furthermore need to add TermURLs for:

- age
- sex
- female
- male
- Hispanic
- MiddleEastern
- Caucasian/African/NativeSouthAmerican
- Caucasian
- Asian/Caucasian
- Asian
- Right
- Left
- Handedness
- dominant_hand
- gender

File: ds000117/participants.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "age": {
+    "LongName": "age",
+    "Units": "year",
+    "TermURL": "TODO"
+  },
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "Units": "TODO",
```
Comment (sappelhoff) [inline ds000117/participants.json:10]:
```suggestion
```

File: ds008/participants.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
```
Comment (sappelhoff) [inline ds008/participants.json:5]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0001894",
```

File: ds101/participants.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
```
Comment (sappelhoff) [inline ds101/participants.json:5]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0001894",
```

File: ds102/participants.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
```
Comment (sappelhoff) [inline ds102/participants.json:5]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0001894",
```

File: ds108/participants.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
```
Comment (sappelhoff) [inline ds108/participants.json:5]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0001894",
```

File: ds110/participants.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
```
Comment (sappelhoff) [inline ds110/participants.json:5]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0001894",
```

File: ds116/participants.json [hunk 7]
```diff
@@ -0,0 +1,68 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
```
Comment (sappelhoff) [inline ds116/participants.json:5]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0001894",
```

File: eeg_ds000117/participants.json [hunk 8]
```diff
@@ -0,0 +1,22 @@
+{
+  "age": {
+    "LongName": "age",
+    "Units": "year",
+    "TermURL": "TODO"
+  },
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
```
Comment (sappelhoff) [inline eeg_ds000117/participants.json:10]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0001894",
```

File: ds000117/participants.json [hunk 9]
```diff
@@ -0,0 +1,35 @@
+{
+  "age": {
+    "LongName": "age",
+    "Units": "year",
+    "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds000117/participants.json:5]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0000011"
```

File: ds008/participants.json [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
+      }
+    }
+  },
+  "age": {
+    "LongName": "age",
+    "Units": "year",
+    "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds008/participants.json:20]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0000011"
```

File: ds009/participants.json [hunk 11]
```diff
@@ -0,0 +1,21 @@
+{
+  "Age": {
+    "LongName": "Age",
+    "Units": "years",
+    "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds009/participants.json:5]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0000011"
```

File: ds101/participants.json [hunk 12]
```diff
@@ -0,0 +1,22 @@
...
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
+      }
+    }
+  },
+  "age": {
+    "LongName": "age",
+    "Units": "year",
+    "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds101/participants.json:20]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0000011"
```

File: ds102/participants.json [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
+      }
+    }
+  },
+  "age": {
+    "LongName": "age",
+    "Units": "year",
+    "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds102/participants.json:24]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0000011"
```

File: ds108/participants.json [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
+      }
+    }
+  },
+  "age": {
+    "LongName": "age",
+    "Units": "year",
+    "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds108/participants.json:20]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0000011"
```

File: ds110/participants.json [hunk 15]
```diff
@@ -0,0 +1,22 @@
...
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
+      }
+    }
+  },
+  "age": {
+    "LongName": "age",
+    "Units": "year",
+    "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds110/participants.json:20]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0000011"
```

File: ds116/participants.json [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
+      }
+    }
+  },
+  "age": {
+    "LongName": "age",
+    "Units": "year",
+    "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds116/participants.json:20]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0000011"
```

File: eeg_ds000117/participants.json [hunk 17]
```diff
@@ -0,0 +1,22 @@
+{
+  "age": {
+    "LongName": "age",
+    "Units": "year",
+    "TermURL": "TODO"
```
Comment (sappelhoff) [inline eeg_ds000117/participants.json:5]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/PATO_0000011"
```

File: ds009/participants.json [hunk 18]
```diff
@@ -0,0 +1,21 @@
...
+  },
+  "Gender": {
+    "LongName": "Gender",
+    "TermURL": "TODO",
```
Comment (sappelhoff) [inline ds009/participants.json:9]:
```suggestion
    "TermURL": "https://schema.org/gender",
```

File: ds009/participants.json [hunk 19]
```diff
@@ -0,0 +1,21 @@
...
+    "Levels": {
+      "F": {
+        "Description": "Female",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds009/participants.json:13]:
```suggestion
        "TermURL": "https://schema.org/Female"
```

File: ds009/participants.json [hunk 20]
```diff
@@ -0,0 +1,21 @@
...
+      },
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds009/participants.json:17]:
```suggestion
        "TermURL": "https://schema.org/Male"
```

File: ds116/participants.json [hunk 21]
```diff
@@ -0,0 +1,68 @@
...
+      "R": {
+        "Description": "Right",
+        "TermURL": "TODO"
+      }
+    }
+  },
+  "ethnicity": {
+    "LongName": "ethnicity",
+    "Description": "TODO",
+    "TermURL": "TODO",
```
Comment (sappelhoff) [inline ds116/participants.json:40]:
```suggestion
    "TermURL": "http://purl.obolibrary.org/obo/HANCESTRO_0004",
```

File: ds116/participants.json [hunk 22]
```diff
@@ -0,0 +1,68 @@
...
+        "Description": "Caucasian",
+        "TermURL": "TODO"
+      },
+      "Caucasian/African/NativeSouthAmerican": {
+        "Description": "Caucasian African NativeSouthAmerican",
+        "TermURL": "TODO"
+      },
+      "Hispanic": {
+        "Description": "Hispanic",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds116/participants.json:60]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/HANCESTRO_0014"
```

File: ds116/participants.json [hunk 23]
```diff
@@ -0,0 +1,68 @@
...
+      },
+      "MiddleEastern": {
+        "Description": "MiddleEastern",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds116/participants.json:64]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/HANCESTRO_0015"
```

File: ds116/participants.json [hunk 24]
```diff
@@ -0,0 +1,68 @@
...
+        "Description": "Asian",
+        "TermURL": "TODO"
+      },
+      "Asian/Caucasian": {
+        "Description": "Asian Caucasian",
+        "TermURL": "TODO"
+      },
+      "Caucasian": {
+        "Description": "Caucasian",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds116/participants.json:52]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/HANCESTRO_0005"
```

File: ds116/participants.json [hunk 25]
```diff
@@ -0,0 +1,68 @@
...
+    }
+  },
+  "ethnicity": {
+    "LongName": "ethnicity",
+    "Description": "TODO",
+    "TermURL": "TODO",
+    "Levels": {
+      "Asian": {
+        "Description": "Asian",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds116/participants.json:44]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/HANCESTRO_0008"
```

File: ds116/participants.json [hunk 26]
```diff
@@ -0,0 +1,68 @@
...
+      },
+      "Asian/Caucasian": {
+        "Description": "Asian Caucasian",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds116/participants.json:48]:
```suggestion
        "Description": "Asian Caucasian"
```

File: ds116/participants.json [hunk 27]
```diff
@@ -0,0 +1,68 @@
...
+      },
+      "Caucasian": {
+        "Description": "Caucasian",
+        "TermURL": "TODO"
+      },
+      "Caucasian/African/NativeSouthAmerican": {
+        "Description": "Caucasian African NativeSouthAmerican",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds116/participants.json:56]:
```suggestion
        "Description": "Caucasian African NativeSouthAmerican"
```

File: ds116/participants.json [hunk 28]
```diff
@@ -0,0 +1,68 @@
...
+      },
+      "R": {
+        "Description": "Right",
+        "TermURL": "TODO"
+      }
+    }
+  },
+  "ethnicity": {
+    "LongName": "ethnicity",
+    "Description": "TODO",
```
Comment (sappelhoff) [inline ds116/participants.json:39]:
```suggestion
```

File: ds116/participants.json [hunk 29]
```diff
@@ -0,0 +1,68 @@
...
+  },
+  "age": {
+    "LongName": "age",
+    "Units": "year",
+    "TermURL": "TODO"
+  },
+  "handedness": {
+    "LongName": "handedness",
+    "Description": "String value indicating handedness",
+    "TermURL": "TODO",
```
Comment (sappelhoff) [inline ds116/participants.json:25]:
```suggestion
    "TermURL": "http://purl.bioontology.org/ontology/SNOMEDCT/57427004",
```

File: ds116/participants.json [hunk 30]
```diff
@@ -0,0 +1,68 @@
...
+    "Levels": {
+      "L": {
+        "Description": "Left",
+        "TermURL": "TODO"
+      },
+      "R": {
+        "Description": "Right",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds116/participants.json:33]:
```suggestion
        "TermURL": "http://purl.bioontology.org/ontology/SNOMEDCT/46669005"
```

File: ds116/participants.json [hunk 31]
```diff
@@ -0,0 +1,68 @@
...
+    "TermURL": "TODO"
+  },
+  "handedness": {
+    "LongName": "handedness",
+    "Description": "String value indicating handedness",
+    "TermURL": "TODO",
+    "Levels": {
+      "L": {
+        "Description": "Left",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds116/participants.json:29]:
```suggestion
        "TermURL": "http://purl.bioontology.org/ontology/SNOMEDCT/87683000"
```

File: ds114/participants.json [hunk 32]
```diff
@@ -0,0 +1,17 @@
...
+    "Description": "TODO",
+    "TermURL": "TODO",
+    "Levels": {
+      "left": {
+        "Description": "left",
+        "TermURL": "TODO"
+      },
+      "right": {
+        "Description": "right",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds114/participants.json:13]:
```suggestion
        "TermURL": "http://purl.bioontology.org/ontology/SNOMEDCT/46669005"
```

File: 7t_trt/participants.json [hunk 33]
```diff
@@ -0,0 +1,21 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
+    "Levels": {
+      "F": {
+        "Description": "Female",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline 7t_trt/participants.json:9]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000383"
```

File: ds000117/participants.json [hunk 34]
```diff
@@ -0,0 +1,35 @@
...
+  },
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "Units": "TODO",
+    "TermURL": "TODO",
+    "Levels": {
+      "F": {
+        "Description": "Female",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds000117/participants.json:15]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000383"
```

File: ds008/participants.json [hunk 35]
```diff
@@ -0,0 +1,22 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
+    "Levels": {
+      "F": {
+        "Description": "Female",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds008/participants.json:9]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000383"
```

File: ds101/participants.json [hunk 36]
```diff
@@ -0,0 +1,22 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
+    "Levels": {
+      "F": {
+        "Description": "Female",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds101/participants.json:9]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000383"
```

File: ds102/participants.json [hunk 37]
```diff
@@ -0,0 +1,26 @@
...
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
+    "Levels": {
+      "D": {
+        "Description": "D",
+        "TermURL": "TODO"
+      },
+      "F": {
+        "Description": "Female",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds102/participants.json:13]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000383"
```

File: ds102/participants.json [hunk 38]
```diff
@@ -0,0 +1,26 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
+    "Levels": {
+      "D": {
+        "Description": "D",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds102/participants.json:9]: what is this supposed to mean?

File: ds108/participants.json [hunk 39]
```diff
@@ -0,0 +1,22 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
+    "Levels": {
+      "F": {
+        "Description": "Female",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds108/participants.json:9]:
```suggestion
        "Description": "Female",
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000383"
```

File: ds110/participants.json [hunk 40]
```diff
@@ -0,0 +1,22 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
+    "Levels": {
+      "F": {
+        "Description": "Female",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds110/participants.json:9]:
```suggestion
        "Description": "Female",
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000383"
```

File: ds116/participants.json [hunk 41]
```diff
@@ -0,0 +1,68 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
+    "Levels": {
+      "F": {
+        "Description": "Female",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds116/participants.json:9]:
```suggestion
        "Description": "Female",
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000383"
```

File: eeg_ds000117/participants.json [hunk 42]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "TODO",
+    "Levels": {
+      "F": {
+        "Description": "Female",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline eeg_ds000117/participants.json:14]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000383"
```

File: 7t_trt/participants.json [hunk 43]
```diff
@@ -0,0 +1,21 @@
...
+      },
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline 7t_trt/participants.json:13]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000384"
```

File: ds000117/participants.json [hunk 44]
```diff
@@ -0,0 +1,35 @@
...
+      },
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds000117/participants.json:19]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000384"
```

File: ds008/participants.json [hunk 45]
```diff
@@ -0,0 +1,22 @@
...
+      },
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds008/participants.json:13]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000384"
```

File: ds101/participants.json [hunk 46]
```diff
@@ -0,0 +1,22 @@
...
+      },
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds101/participants.json:13]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000384"
```

File: ds102/participants.json [hunk 47]
```diff
@@ -0,0 +1,26 @@
...
+      },
+      "F": {
+        "Description": "Female",
+        "TermURL": "TODO"
+      },
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds102/participants.json:17]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000384"
```

File: ds108/participants.json [hunk 48]
```diff
@@ -0,0 +1,22 @@
...
+      },
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds108/participants.json:13]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000384"
```

File: ds110/participants.json [hunk 49]
```diff
@@ -0,0 +1,22 @@
...
+      },
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds110/participants.json:13]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000384"
```

File: ds116/participants.json [hunk 50]
```diff
@@ -0,0 +1,68 @@
...
+      },
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds116/participants.json:13]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000384"
```

File: eeg_ds000117/participants.json [hunk 51]
```diff
@@ -0,0 +1,22 @@
...
+      },
+      "M": {
+        "Description": "Male",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline eeg_ds000117/participants.json:18]:
```suggestion
        "TermURL": "http://purl.obolibrary.org/obo/PATO_0000384"
```

File: ds114/participants.json [hunk 52]
```diff
@@ -0,0 +1,17 @@
+{
+  "dominant_hand": {
+    "LongName": "dominant_hand",
+    "Description": "TODO",
+    "TermURL": "TODO",
+    "Levels": {
+      "left": {
+        "Description": "left",
+        "TermURL": "TODO"
```
Comment (sappelhoff) [inline ds114/participants.json:9]:
```suggestion
        "TermURL": "http://purl.bioontology.org/ontology/SNOMEDCT/87683000"
```

File: ds114/participants.json [hunk 53]
```diff
@@ -0,0 +1,17 @@
+{
+  "dominant_hand": {
+    "LongName": "dominant_hand",
+    "Description": "TODO",
+    "TermURL": "TODO",
```
Comment (sappelhoff) [inline ds114/participants.json:5]:
```suggestion
    "LongName": "dominant_hand"
```

File: ds102/participants.json [hunk 54]
```diff
@@ -0,0 +1,25 @@
+{
+  "sex": {
+    "LongName": "sex",
+    "Description": "String value indicating phenotypical sex.",
+    "TermURL": "http://purl.obolibrary.org/obo/PATO_0001894",
+    "Levels": {
+      "D": {
+        "Description": "D"
```
Comment (sappelhoff) [inline ds102/participants.json:8]: I have removed "TermURL" here for now. If somebody knows what this means, please feel free to clarify.

**Comment by sappelhoff** [comment]
Okay:

- passes on master ✅ 
- adds correct TermURLs to many examples, exemplifying the new feature ✅ 
- I dropped everything that was unclear but wasn't really needed ✅ 
    - feel free to add later: for example what's the "D" sex?
    - or a TermURL for `Caucasian/African/NativeSouthAmerican`
- better done than perfect ✅ 

Will merge.

Thanks for the start @Remi-Gau and thanks for the support @surchs!

**Pull request merged**